runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -218,18 +218,21 @@ public Dimension render(Graphics2D graphics)
 
 			if (config.priceDisplayMode() == PriceDisplayMode.BOTH)
 			{
-				if (item.getGePrice() > 0)
+				if (item.getItemId() == 995)
```

Comment:
Why is this specific only for PriceDisplayMode.BOTH? Also, shouldn't this condition be inversed? Also, use ItemID.COINS_995